core/thread/inc/TWin32Mutex.h

```diff
@@ -36,6 +36,8 @@ friend class TWin32Condition;
 private:
    CRITICAL_SECTION fCritSect;
 
+   constexpr static int kIsRecursive = BIT(14);
```

Comment:
Since this is use with fBits, this must be in an enum name EStatusBits (this allows automatic checking for overlaps). Thanks.

Reply:
As previously said, this code is copied over from TPosixMutex...